vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -660,9 +674,8 @@ bool BatPublishers::isEligibleForContribution(const ledger::PublisherInfo& info)
   if (isExcluded(info.id, info.excluded) || (!state_->allow_non_verified_ && !isVerified(info.id)))
     return false;
 
-  return info.score > 0 &&
-    info.duration >= state_->min_publisher_duration_ &&
-    info.visits >= state_->min_visits_;
+  return info.duration >= state_->min_publisher_duration_ &&
```

Comment:
we need to remove this check for score as old versions had negative values, which was now fixed

Reply:
I would keep it as it. This code was not changed and would like to minimize  changes as much as possible as this goes into release.